.github/workflows/weekly_ci.yml

```diff
@@ -0,0 +1,67 @@
+# yamllint disable rule:line-length
+name: Weekly CI trigger
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 0 * * 6'
+env:
+  BRANCH_NAME: "weekly-ci"
+  COMMIT_MESSAGE: "Weekly CI check"
+jobs:
+  weekly_ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete the previous branch if exists
```

Comment:
@ukulekek this task fails in the initial action run because the branch doesn't exist yet: https://github.com/linux-system-roles/mssql/actions/runs/3253985191/jobs/5341776134